contracts/FlowIDTableStaking.cdc

```diff
@@ -868,6 +868,11 @@ pub contract FlowIDTableStaking {
 
         let nodeRecord = FlowIDTableStaking.borrowNodeRecord(nodeID)
 
+        assert (
+            nodeRecord.role != UInt8(5),
+            message: "Cannot register a delegator for an access node"
```

Comment:
because access node don't get reward?

Reply:
Yes, I just want to prevent delegators from delegating to a place where they won't get rewards